Client/Application/AdjustIntegration.swift

```diff
@@ -192,4 +197,23 @@ extension AdjustIntegration: AdjustDelegate {
         return true
     }
 
+    private func migratePathComponentInDocumentsDirectory(_ pathComponent: String, to destinationSearchPath: FileManager.SearchPathDirectory) {
+        guard let oldPath = try? FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: false).appendingPathComponent(pathComponent).path, FileManager.default.fileExists(atPath: oldPath) else {
+            return
+        }
+
+        print("Migrating \(pathComponent) from ~/Documents to \(destinationSearchPath)")
```

Comment:
print statements wont show up in logs. Should we just remove them? If we want to show this in logs we can use xcglogger